doc/en/user/source/services/wfs/webadmin.rst

```diff
@@ -121,4 +121,17 @@ Option :guilabel:`Override MIME Type` allows the selection of the MIME type that
 
 .. figure:: img/services_WFS_mimetype.png
 
-The available MIME types are: ``application/gml+xml; version=3.2``, ``text/xml; subtype=gml/3.2`` and ``text/xml``. 
\ No newline at end of file
+The available MIME types are: ``application/gml+xml; version=3.2``, ``text/xml; subtype=gml/3.2`` and ``text/xml``. 
+
+Configure XML Entity Expansion limit on WFS XML readers
+-------------------------------------------------------
+
+By default WFS XML readers sets Entity Expansion limit to 100, but it can be configured via the ``org.geoserver.wfs.xml.entityExpansionLimit`` system property.
```

Comment:
Since it's using GeoServerExtensions.getProperty, it can also be an environment variable or a servlet context property, see https://github.com/geoserver/geoserver/blob/70c8e179f669db3b18fb215e1933744950a43283/src/platform/src/main/java/org/geoserver/platform/GeoServerExtensions.java#L361

Reply:
Also, a simpler approach would have been to put the system variable reading in the geotools code directly, and then document that name (we have a number of geotools specific system variables, no requirement for them to start with org.geoserver to end up in the geoserver docs). That would simplify the PR.